python_modules/dagster/dagster/core/asset_defs/asset_group.py

```diff
@@ -106,7 +107,7 @@ def __new__(
             )
         # In the case of collisions, merge_dicts takes values from the dictionary latest in the list, so we place the user provided resource defs after the defaults.
         resource_defs = merge_dicts(
-            {"root_manager": root_manager, "io_manager": default_job_io_manager},
+            {"root_manager": root_manager, "io_manager": fs_asset_io_manager},
```

Comment:
can't comment on the line where `default_job_io_manager` is imported in `__new__` but that import seems unnecessary now